tools/test-oot-build.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -euo pipefail
```

Comment:
should we move this file to tools/ci/cibuild-oot.sh? since it's specially designed for ci.

Reply:
good point, will move this